client/packages/ecore-property-view/src/browser/widget-provider.ts

```diff
@@ -104,6 +115,9 @@ export class EcoreGlspPropertyViewWidgetProvider extends JsonFormsPropertyViewWi
                 this.jsonFormsWidget.updateModelServerWidgetData(command.feature, newValue);
                 this.currentPropertiesCore = this.jsonFormsWidget.currentJsonFormsCore;
             }
+        } else if (relativeRef.split("#")[0] === this.currentModelUri && command.type === "remove") {
```

Comment:
We should avoid parsing string values like this. Since semanticUri is expected to be a URI, there is probably a built-in URI type that we can use

Reply:
Good point, I reused the URI type with its path and fragment for the checks.